### ISSUE TITLE
Edit to "Add Rackspace Email mailboxes"

### ISSUE THREAD

**Issue by misskristenbrown** [rackerlabs/rackspace-how-to#1730]
https://github.com/rackerlabs/rackspace-how-to/blob/master/content/rackspace-email/add-rackspace-email-mailboxes.md

1. Current Section without Changes:  No mention of how to add mailboxes when "Add Mailbox" button is not enabled.
*Proposed Changes:* Add in a section to cover instructions for customers whose "Add Mailboxes" button is greyed out in its own textbox and/or as a block indent. Please see the following text for the proposed addition of text and screenshot: 

"NOTE: If your “Add Mailbox” button is “greyed out,” then you will need to first add the necessary license(s) to your account. 
 
"Add license(s)
1.	Log in to the Cloud Office Control Panel using your Rackspace Cloud Office Admin ID and password.
2.	In the Go to section list, choose Rackspace Email.
3.	Click the Email Accounts tab.
4.	If you have multiple domains, select the appropriate domain name.
5.	Click Upgrade this plan.
6.	Within the appropriate section, add the desired number of mailboxes.
7.	Click Continue.
8.	Review the Order Summary and purchase the licenses by clicking Complete Order."

![addmailbox_greyedout](https://cloud.githubusercontent.com/assets/24738359/21472189/b326c3a4-ca92-11e6-99c6-0863e3e25d14.png)

2. Current Section without Changes: From all sections, first instruction, "Log in to the Cloud Office Control Panel using your Rackspace username and password." is too vague; customers may confuse their "Rackspace username and password" with their email account or MyCustomer portal credentials. 
*Proposed Changes:* Since a customer may have multiple Rackspace usernames/passwords, clarify the sentence, "Log in to the Cloud Office Control Panel using your Rackspace Cloud Office Admin ID and password."

3. Current Section without Changes: From section "Add a Single Mailbox," number six, directions for User Name reads: "Enter the first part of the email address. For example, myname@mydomain.com."
*Proposed Changes:* Clarify "the first part" and the example. "User Name: Enter the first part of the email address – the part before the @ symbol. For example: “myname” (without quotes) from myname@mydomain.com"

4. Current Section without Changes: From section "Add Multiple Mailboxes," there is a screenshot which displays the button "Choose File"
*Proposed Changes:* Replace screenshot with attached, updated screenshot which displays the button "Browse..."
![addmailboxes](https://cloud.githubusercontent.com/assets/24738359/21472272/751105da-ca96-11e6-8dd0-15970d56f460.png)

5. Current Section without Changes: From section "Add Multiple Mailboxes," number seven, "In the Import File section, click the Choose File button and locate and select the CSV or Excel file that you created."
*Proposed Changes:* Replace "Choose File" with "Browse..." since this is how it appears in the Cloud Office Control Panel. 

See attached document with my proposed changes:
[Add Rackspace Email mailboxes.docx](https://github.com/rackerlabs/rackspace-how-to/files/672274/Add.Rackspace.Email.mailboxes.docx)

**Comment by nimbinatus** [comment]
Thanks, @misskristenbrown! We will pull this update into our next sprint.

**Comment by nimbinatus** [comment]
@misskristenbrown, when we check the steps on an active Rackspace Email account's control panel, we are still seeing a `Choose File` button instead of a `Browse` button for the 4th change. Is that image from a production environment?

**Comment by misskristenbrown** [comment]
Hello Laura,
I have looked into this and I see it is a browser-related discrepancy. To clarify, the image was not from a production environment, but rather from Mozilla Firefox. I checked in Google Chrome, and I see it does say Choose File… Therefore, could we update it to be browser-neutral and say “Browse” or “Choose File…” No need to update the images to include both.
Thanks.

**Comment by nimbinatus** [comment]
@misskristenbrown, thanks for the info! I'll do as you suggested.

**Comment by misskristenbrown** [comment]
Originally, I had suggested that the "Add licenses" section be more upfront for the following reason:
If a customer has exhausted their licenses, the Add Mailbox button is greyed out. 
Therefore, in many cases, adding licenses is step one. 
Is there a way we can better lay out the steps? In my example, I included it as an "asterisked" note. 
I have attached it again for review. 
[Add Rackspace Email mailboxes.docx](https://github.com/rackerlabs/rackspace-how-to/files/812130/Add.Rackspace.Email.mailboxes.docx)
